dotcom-rendering/src/palette.ts

```diff
@@ -2876,7 +2876,81 @@ const syndicationButtonBorder: PaletteFunction = ({ design, theme }) => {
 			return sourcePalette.neutral[86];
 	}
 };
+const mostViewedHeadlineLight = (): string => sourcePalette.neutral[7];
+const mostViewedHeadlineDark = (): string => sourcePalette.neutral[86];
```

Comment:
```suggestion
const mostViewedHeadlineLight: PaletteFunction = () => sourcePalette.neutral[7];
const mostViewedHeadlineDark: PaletteFunction = () => sourcePalette.neutral[86];
```

I think this would make @mxdvl happy!

Reply:
I’m only trying to help us figuring out errors early! <3